yaml_edit/test/test_case.dart

```diff
@@ -225,9 +225,9 @@ Map _getValueFromYamlMap(YamlMap yamlMap) {
 dynamic _getValueFromYamlNode(YamlNode node) {
   switch (node.runtimeType) {
     case YamlList:
-      return _getValueFromYamlList(node);
+      return _getValueFromYamlList(node as YamlList);
     case YamlMap:
-      return _getValueFromYamlMap(node);
+      return _getValueFromYamlMap(node as YamlMap);
     default:
       return node.value;
```

Comment:
This whole section could have been changed to:
```dart
  if (node is YamlList) {
    return _getValueFromYamlList(node);
  }
  if (node is YamlMap) {
    return _getValueFromYamlMap(node);
  }
  return node.value;
```
...but I'm not sure of the implications of those changes yet.

Reply:
`node is YamlList` means that `node` is an instance of some class that extends / implements `YamlList`.

However, `node.runtimeType == YamlList` only happens if `node` is an instance of  `YamlList`.

Look for other implementations of `YamlList` that might explain why this works this way.
Could also just be something obscure, you can try to change to test and see if it makes a difference.

If we refactor this, let's do it a separate PR (if it's null-safety unrelated).

Reply:
I see. Got it.